patches/services/fabricv4/20240419-add-package-alias-to-custom-package-param-precision-time.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/services/fabricv4/test/api_precision_time_test.go b/services/fabricv4/test/api_precision_time_test.go
+index 2fb4344..bcfca38 100644
+--- a/services/fabricv4/test/api_precision_time_test.go
++++ b/services/fabricv4/test/api_precision_time_test.go
+@@ -71,7 +71,7 @@ func Test_fabricv4_PrecisionTimeApiService(t *testing.T) {
+ 	t.Run("Test PrecisionTimeApiService GetTimeServicesPackageByCode", func(t *testing.T) {
+ 		t.Skip("skip test") // remove to run test
+ 
+-		var packageCode GetTimeServicesPackageByCodePackageCodeParameter
++		var packageCode openapiclient.GetTimeServicesPackageByCodePackageCodeParameter
```

Comment:
The post-patch is good for now.  This seems like a bug in the Go generator, so it would be good to find a more comprehensive fix before we build up a library of similar patches.  This may be as "easy" as customizing the test template but that requires more time to figure out.

Reply:
That's a good thought. I can look into this the next time around. Thank you.